.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 
 node_js:
-  - "6"
+  - 10
```

Comment:
if we still support Node 8 then we should probably test on Node 8 😉

Reply:
Yeah, good point - this is essentially copied from simplabs/ember-simple-auth, any idea why we do it like this over there?

* https://github.com/simplabs/ember-simple-auth/blob/master/.travis.yml#L8
* https://github.com/simplabs/ember-simple-auth/blob/master/package.json#L100

Reply:
I dropped support for Node 8 as well - no need to keep that really.

Reply:
looks like I missed that when I looked at https://github.com/simplabs/ember-simple-auth/pull/1956